tests/integration/telemetry/stats/prometheus/customizemetrics/testdata/attributegen_envoy_filter.yaml

```diff
@@ -56,6 +77,9 @@ spec:
                       }]
                     }
                 vm_config:
-                  runtime: envoy.wasm.runtime.null
+                  runtime: envoy.wasm.runtime.v8
                   code:
-                    local: { inline_string: "envoy.wasm.attributegen" }
+                    remote:
+                      http_uri:
+                        uri: https://storage.googleapis.com/istio-build/proxy/attributegen-{{ .ProxySHA }}.wasm
+                        timeout: 10s
```

Comment:
just curious did we test how long it usually takes to do a remote load?

Reply:
It is actually pretty fast. I don't have exact stats on only remote load, but envoy takes about 2s to become ready when loading attributes gen module remotely.